clients/web/src/stylesheets/body/systemConfig.styl

```diff
@@ -5,6 +5,18 @@
   .bootstrap-switch-handle-off
       word-wrap normal
 
+#g-core-banner-color-container
+  display block
+
+  #g-core-banner-color
+    display inline
+    max-width 15%
```

Comment:
On larger displays, making this element a percentage of the overall viewport width looks strange:
![image](https://user-images.githubusercontent.com/1282879/30674748-32410d4e-9e4a-11e7-8de0-bfeccee1bf74.png)

Can you change it to use a fixed number of pixels instead? I'm fine if it continues to use either `max-width` or `width` (depending on whether you want to allow it to shrink or not).